### ISSUE TITLE
About the network initialization during training

### ISSUE THREAD

**Issue by Mr-watermelon** [brijeshiitg/Pytorch-implementation-of-SRNet#6]
Hi, author, I would like to ask if your network has been initialized as stated in the paper?

**Comment by Mr-watermelon** [comment]
Hello, I have used data enhancement, but it still cannot be converged. Acc is always 0.5 and loss is always 0.69. Why?

**Comment by Mr-watermelon** [comment]
Hello, author.Thanks for your model source code, I have found my problem and solved it.thank you

**Comment by LeeeLiu** [comment]
> Hello, author.Thanks for your model source code, I have found my problem and solved it.thank you

Hi, friends,  I  meet  the  same  problem “ ACC is always 0.5 ”
How did you solve it？

**Comment by Mr-watermelon** [comment]
> > Hello, author.Thanks for your model source code, I have found my problem and solved it.thank you
> 
> Hi, friends, I meet the same problem “ ACC is always 0.5 ”
> How did you solve it？


1. model 's  layer have used log_softmax, so loss fuction should use nll_loss .
2.Data should be fed into the network in pairs, meaning that Cover/Stego needs to be paired

**Comment by LeeeLiu** [comment]
谢谢你的回复！我试试
（我使用了作者建议的“数据增强”，但是效果还是不收敛）


 
---Original---

**Comment by davidggz** [comment]
> > > Hello, author.Thanks for your model source code, I have found my problem and solved it.thank you
> > 
> > 
> > Hi, friends, I meet the same problem “ ACC is always 0.5 ”
> > How did you solve it？
> 
> 1. model 's  layer have used log_softmax, so loss fuction should use nll_loss .
>    2.Data should be fed into the network in pairs, meaning that Cover/Stego needs to be paired

Thank you for your response! I managed to get 80% accuracy on test just by training using the Cover/Stego pairing in each batch. It's quite strange that works in this case. The only problem that I'm seeing is that the training is hard to replicate since it always get stuck for a long period into 0.69 loss.

**Comment by anzhitinglan3** [comment]
> > Hi, friends, I meet the same problem “ ACC is always 0.5 ”
> > How did you solve it？
> 
> 1. model 's  layer have used log_softmax, so loss fuction should use nll_loss .
>    2.Data should be fed into the network in pairs, meaning that Cover/Stego needs to be paired

Hello, I would like to ask whether the ACC you are talking about here is training or testing?

**Comment by anzhitinglan3** [comment]
> > > Hello, author.Thanks for your model source code, I have found my problem and solved it.thank you
> > 
> > 
> > Hi, friends, I meet the same problem “ ACC is always 0.5 ”
> > How did you solve it？
> 
> 1. model 's  layer have used log_softmax, so loss fuction should use nll_loss .
>    2.Data should be fed into the network in pairs, meaning that Cover/Stego needs to be paired

Can you share the code you tested? Thank you
